### PR TITLE
chore(pacmak): one snapshot file per language

### PR DESCRIPTION
**Pull request by RomainMuller**
Hopefully this results in smaller diffs when significant changes are
performed on the code generators, and they can more reliably be reviewed
in GitHub.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by RomainMuller** [comment]
Marked with `pr/do-not-merge` to avoid forcing merge conflicts on #1951 (I'd rather suffer the merge conflicts myself than inflict them to someone else), will unblock once that is merged.

**Review by rix0rrr — APPROVED** [review]

**Pull request merged**
